### PR TITLE
Feature/update start up docs

### PR DESCRIPTION
**Pull request by LindsayYoung**
Updating docs to convey the current set-up structure
got rid of the vagrant instructions, since we have not maintained them
got rid of `$` to add improve docs as suggested by a helpful user

@emileighoutlaw and @jmcarp if you can check or improve the copy, that would be great. 

Thanks to @patphongs for helping us troubleshoot this. Pat, does this look better?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,20 +19,24 @@ Are you interested in seeing how much money a candidate raised? Or spent? How mu
...
 
 ### All repositories
 - [FEC](https://github.com/18F/fec): a general discussion forum. We [compile feedback](https://github.com/18F/fec/issues) from betaFEC’s feedback widget here, and this is the best place to submit general feedback.
 - [openFEC](https://github.com/18F/openfec): betaFEC’s API
+- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
 - [openFEC-web-app](https://github.com/18f/openfec-web-app): the betaFEC web app for exploring campaign finance data
 - [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
 - [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
+- [proxy](https://github.com/18F/fec-proxy): this is a light weight app that coordinates the paths between the web app and cms
+- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
```
Comment (jmcarp) [inline README.md]: Looks like this line got added twice.

File: README.md [hunk 2]
```diff
@@ -19,20 +19,24 @@ Are you interested in seeing how much money a candidate raised? Or spent? How mu
...
+
 
 ## Get involved
-We’re thrilled you want to get involved! 
-- Read our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit a pull request.
+We’re thrilled you want to get involved!
+- Read our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit an issue or pull request.
```
Comment (jmcarp) [inline README.md]: Looks like this would read "Then, file an issue or submit an issue or pull request". Why the change?

File: README.md [hunk 3]
```diff
@@ -55,41 +54,85 @@ Prior to running, ensure you have the following requirements installed:
...
+
+
+## Deployment
 
 ##### Likely only useful for 18F team members
 To deploy to Cloud Foundry, run `invoke deploy`. The `deploy` task will attempt to detect the appropriate
 Cloud Foundry space based the current branch; to override, pass the optional `--space` flag:
 
-    $ invoke deploy --space dev
+```
```
Comment (jmcarp) [inline README.md]: Why change from indents to code fencing? AFAICT, they do the same thing. Just curious.

File: README.md (hunk 2 above)
Comment (LindsayYoung) [inline README.md]: haha, disregard. Sometimes, I misread things when there are a lot of links.

File: README.md (hunk 1 above)
Comment (LindsayYoung) [inline README.md]: I meant to move it up, rather than re-add it, since it is closely related to this repo. I'll fix.

File: README.md (hunk 3 above)
Comment (LindsayYoung) [inline README.md]: Just getting rid of the $, wasn't confidant the indents would all have the code markup without them if they were not fenced.

Comment (jmcarp) [inline README.md]: It's helpful to write markdown with some kind of previewing tool--for sublime, maybe something like https://github.com/revolunet/sublimetext-markdown-preview. Then you'll be able to see what the formatted output will look like without referring to docs or guessing.

**Comment by emileighoutlaw** [comment]
This looks great. Still making a couple notes. I'll post again when I'm done. Thanks for doing this, Lindsay. These are awesome updates. :)

**Comment by emileighoutlaw** [comment]
Kk! Made all my comments!

**Comment by LindsayYoung** [comment]
Thank you @emileighoutlaw I added your revisions!

**Comment by emileighoutlaw** [comment]
LGTM!
